filebrowser/main.tf

```diff
@@ -20,7 +20,7 @@ data "coder_workspace_owner" "me" {}
 
 variable "agent_name" {
   type = string
-  default = ""
+  default = "main"
```

Comment:
```suggestion

```
let us remove default to make `agent_name` a required input. What do you think?

Reply:
Since a suggestion for "main" is given in the readme. This can be an required input.